package.nls.json

```diff
@@ -8,7 +8,10 @@
     "azIntegrationAccounts.showExplorerDescription": "Show or hide the Azure Integration Accounts Explorer",
     "azIntegrationAccounts.showSavePrompt": "Prompt the user before saving and updating a Logic App",
     "azIntegrationAccounts.viewProperties": "View properties",
-    "azLogicApps.createLogicApp": "Create",
+    "azLogicApps.addBuildDefinitionToProject": "Add Build Definition to Project...",
```

Comment:
None of the other items end in an ellipses, should this be consistent? Aka, should we either remove this one, add it to other ones, or is this a special case?

Reply:
The rule of thumb is that if it presents a dialog or wizard of some sort, show ellipses.  Otherwise, do not.  In this case, we're showing a wizard.